Cargo.toml

```diff
@@ -93,6 +93,7 @@ base64 = "0.10"
 safemem = "0.3"
 cop_datatype = { path = "components/cop_datatype" }
 smallvec = { version = "0.6", features = ["union"] }
+more-asserts = "0.1"
```

Comment:
It seems a dev-dependency.

Reply:
I think it may be useful elsewhere in future, for example, replacing existing `assert!(xxx >= yyy)` in code base (not only tests), so that error message can be more friendly.